jerry-core/jerry.h

```diff
@@ -36,8 +36,7 @@ extern "C"
 #define JERRY_FLAG_SHOW_OPCODES           (1u << 0) /**< dump byte-code to stdout after parse */
 #define JERRY_FLAG_MEM_STATS              (1u << 1) /**< dump memory statistics */
 #define JERRY_FLAG_MEM_STATS_PER_OPCODE   (1u << 2) /**< dump per-instruction memory statistics during execution
-                                                     *   (in the mode full GC is performed after execution of each
-                                                     *   opcode handler) */
+                                                     *   FIXME: Remove. */
```

Comment:
I wasn't sure whether I should renumber all unrelated flags or leave an unused hole in the bits. But I'd go for the renumbering then.